### DISCUSSION TITLE
Broadcast different length

### DISCUSSION THREAD

**Discussion by NorbertGarnys** [john30/ebusd#611] [Q&A]
Hi,

For the record I have the Ariston CLAS PREMIUM EVO boiler. I'm trying to decode the broadcast messages.
What I notices is that they they don't have strict structure i.e.

2022-06-13 00:41:57.750 <37fe**2010**05**6147**c2010020
2022-06-12 22:41:07.093 <37fe**2010**09**6147**c20100**c04b**0100d8
2022-06-12 22:39:16.956 <37fe**2010**04**c04b**010030

It's common that in 2010 and 2020 PBSB it can be more than one ID 
2022-06-12 22:40:10.136 <37fe**2020**10**6172**5e01**6173**5e01**6174**5e01**6175**5e01e9
Looks a bit like a register dump.
And question is there a way to decode that with csv configuration file ?

**Comment by IvanHorban** [comment]
Hi. I have the ariston cares xc 24  boiler and I also can't decrypt messages. I found this https://github.com/komw/ariston-bus-bridgenet-ebusd, but I don't know how to use it.

**Comment by john30** [comment]
just pick up the [ariston.csv](https://github.com/komw/ariston-bus-bridgenet-ebusd/blob/master/configuration/ariston.csv) file, put into a folder on your host, and point ebusd to that directory using the `--configpath=...` option

**Comment by IvanHorban** [comment]
Thanks for helping. Now I get information from boiler. But I have some problem. Outside temperature 16.7 °C, but I get 167 °C. The same with the heating_pre_temp 580 °C. How to fix it?

**Reply by john30** [reply]
then the definition of the message is complete. obviously both miss a divider of 10
